app/components/page-components/Chatbot/windows/ChatbotDefaultCommandWindow.vue.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, Prop } from 'vue-property-decorator';
+import ChatbotWindowsBase from 'components/page-components/Chatbot/windows/ChatbotWindowsBase.vue';
+import TextInput from 'components/shared/inputs/TextInput.vue';
+import TextAreaInput from 'components/shared/inputs/TextAreaInput.vue';
+import ListInput from 'components/shared/inputs/ListInput.vue';
+import { cloneDeep } from 'lodash';
+import ChatbotAliases from 'components/page-components/Chatbot/shared/ChatbotAliases.vue';
+
+import {
+  IDefaultCommand,
+} from 'services/chatbot/chatbot-interfaces';
+
+
+import {
+  IListMetadata,
+  ITextMetadata,
+} from 'components/shared/inputs/index';
+
+
+interface IDefaultCommandMetadata {
+  command: ITextMetadata;
+  response: ITextMetadata;
+  new_alias: ITextMetadata;
+  success_response: ITextMetadata;
+  failed_response: ITextMetadata;
+  enabled_response: ITextMetadata;
+  disabled_response: ITextMetadata;
+  response_type: IListMetadata<string>;
+}
+
+
+@Component({
+  components: {
+    TextInput,
+    TextAreaInput,
+    ListInput,
+    ChatbotAliases
+  }
+})
+export default class ChatbotDefaultCommandWindow extends ChatbotWindowsBase {
+  editedCommand: IDefaultCommand = null;
+
+  tabs: { name: string; value: string }[] = [
+    {
+      name: 'General',
+      value: 'general'
+    },
+    {
+      name: 'Advanced',
+      value: 'advanced'
+    }
+  ];
+
+  selectedTab: string = 'general';
+
+  mounted() {
+    this.editedCommand = cloneDeep(this.defaultCommandToUpdate);
+  }
+
+  get isLinkProtectionPermitCommand() {
+    return this.defaultCommandToUpdate.slugName === 'link-protection'
+    && this.defaultCommandToUpdate.commandName === 'permit';
+  }
+
+  get defaultCommandToUpdate() {
+    return this.chatbotCommonService.state.defaultCommandToUpdate;
+  }
+
+  // metadata
+  get metadata() {
+    let metadata: IDefaultCommandMetadata = {
+      command: this.stringMetadata(
```

Comment:
`import { metadata } from 'components/widgets/inputs';`
`this.stringMetadata()` -> `metadata.string()`